components/style/keyframes.rs

```diff
@@ -70,28 +72,7 @@ impl KeyframeSelector {
 #[cfg_attr(feature = "servo", derive(HeapSizeOf))]
 pub struct Keyframe {
     pub selector: KeyframeSelector,
-    pub declarations: Arc<Vec<PropertyDeclaration>>,
-}
-
-impl Keyframe {
-    pub fn parse(context: &ParserContext, input: &mut Parser) -> Result<Keyframe, ()> {
-        let percentages = try!(input.parse_until_before(Delimiter::CurlyBracketBlock, |input| {
-            input.parse_comma_separated(|input| KeyframePercentage::parse(input))
-        }));
-        let selector = KeyframeSelector(percentages);
-
-        try!(input.expect_curly_bracket_block());
-
-        let declarations = input.parse_nested_block(|input| {
-            Ok(parse_property_declaration_list(context, input))
-        }).unwrap();
-
-        // NB: Important declarations are explicitely ignored in the spec.
-        Ok(Keyframe {
-            selector: selector,
-            declarations: declarations.normal,
-        })
-    }
+    pub declarations: Arc<Vec<(PropertyDeclaration, Importance)>>,
```

Comment:
`style::animation::compute_style_for_animation_step` crates a `style::selector_matching::DeclarationBlock` struct from this. That struct contains `Arc<Vec<(PropertyDeclaration, Importance)>>`. So having dummy `Importance::Normal` values in `Keyframe` enables doing this by cloning an `Arc` (incrementing a refcount) rather than creating a new `Vec`.

I’ve added a doc-comment saying this.